e2e/scripts/shared.ts

```diff
@@ -132,3 +134,40 @@ export const buildAndGetResults = async (
     isSuccess: Boolean(rsbuildInstance),
   };
 };
+
+export const buildAndGetAllResults = async (
```

Comment:
Can we combine this function with `buildAndGetResults` as `type: all`to reduce repeated code?